open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/server/AnalyticsModelingRestServices.java

```diff
@@ -0,0 +1,161 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.analyticsmodeling.server;
+
+import java.util.List;
+
+import org.odpi.openmetadata.accessservices.analyticsmodeling.ffdc.exceptions.AnalyticsModelingCheckedException;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.model.ResponseContainerDatabase;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.model.ResponseContainerDatabaseSchema;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.model.ResponseContainerModule;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.model.ResponseContainerSchemaTables;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.AnalyticsModelingOMASAPIResponse;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.DatabasesResponse;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.ErrorResponse;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.ModuleResponse;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.SchemaTablesResponse;
+import org.odpi.openmetadata.accessservices.analyticsmodeling.responses.SchemasResponse;
+import org.odpi.openmetadata.adminservices.configuration.registration.AccessServiceDescription;
+import org.odpi.openmetadata.commonservices.ffdc.RESTCallLogger;
+import org.odpi.openmetadata.commonservices.ffdc.RESTCallToken;
+import org.odpi.openmetadata.commonservices.ffdc.RESTExceptionHandler;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Server-side implementation of the Analytics Modeling OMAS interface for modeling.
+ */
+public class AnalyticsModelingRestServices {
+
+	AnalyticsModelingInstanceHandler instanceHandler = new AnalyticsModelingInstanceHandler();
+
+	private static RESTCallLogger restCallLogger = new RESTCallLogger(LoggerFactory.getLogger(AnalyticsModelingRestServices.class),
+			AccessServiceDescription.ANALYTICS_MODELING_OMAS.getAccessServiceFullName());
+	private RESTExceptionHandler restExceptionHandler = new RESTExceptionHandler();
+
+	/**
+	 * Get databases available on the server for the user.
+	 * 
+	 * @param serverName of the server.
+	 * @param userId     of the user.
+	 * @return list of databases for the requested server/user.
+	 */
+	public AnalyticsModelingOMASAPIResponse getDatabases(String serverName, String userId) {
+
+		String methodName = "getDatabases";
+		AnalyticsModelingOMASAPIResponse ret;
+		RESTCallToken token = restCallLogger.logRESTCall(serverName, userId, methodName);
+
+		try {
+			DatabasesResponse response = new DatabasesResponse();
+			List<ResponseContainerDatabase> databases = instanceHandler
+					.getDatabaseContextHandler(serverName, userId, methodName).getDatabases();
```

Comment:
I can not see 1) where the security verifier is called to check the user has access to the asset - or the supportedZones are checked.